src/coreclr/vm/jithelpers.cpp

```diff
@@ -4214,6 +4216,59 @@ HCIMPLEND
 
 /*************************************************************/
 
+#ifdef FEATURE_EH_FUNCLETS
+HCIMPL1(void, IL_ThrowNew,  Object* obj)
```

Comment:
Diagnostic tools have `IL_Throw` symbol name hard-coded in number of places, e.g. here:
https://microsoft.visualstudio.com/OS/_search?action=contents&text=IL_Throw&type=code&lp=code-Project&filters=ProjectFilters%7BOS%7DRepositoryFilters%7Bos.2020%7D&pageSize=25&includeFacets=false&result=DefaultCollection/OS/os.2020/GBofficial/main//sdktools/debuggers/exts/badev/extdll/uanalysis.cpp

This rename is going to require chasing down all these places and fixing them up, I am not sure that it is worth it.

Reply:
Reverted